src/components/CommentsCard.jsx

```diff
@@ -0,0 +1,31 @@
+import moment from "moment";
+import { useState, useEffect } from "react";
+import { getComments } from "../utils/api";
+
+export const CommentsCard = ({ review_id }) => {
```

Comment:
The name of this component is a bit misleading

"Card" usually means one - this is a list!